src/figuresClasses.ts

```diff
@@ -1,19 +1,62 @@
-export interface Figure {
+type Shape = 'triangle' |'circle' | 'rectangle';
+type Color = 'red' | 'green' | 'blue';
 
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
-
+  shape: Shape = 'triangle';
+
+  constructor(public color: Color,
+    public a: number,
```

Comment:
please, fix formatting here... public color: Color, has to be in a new line as well.
also dont forget about bracket in the end

correct:
```
constructor(
  public color: Color,
  public a: number,
  public b: number,
  public c: number,
) {
  ```